config.yml

```diff
@@ -8,7 +8,8 @@ aml_experiment_details:
 
 azure_iot:
   hostname: "iothub-oor-ont-weu-itr-01.azure-devices.net"
-  device_id: "test-diana"
+  # hostname: "iothub-oor-prd-weu-ggi-01.azure-devices.net"
```

Comment:
why is this commented?

Reply:
That was just to have both hostnames there for easy switching between dev and prod. Otherwise I had to look up the correct hostname each time :)